### PR TITLE
Improve issue and pr template. (bunq/sdk_java#56)

### PR DESCRIPTION
**Pull request by OGKevin**
Closes bunq/sdk_java#56

### CODE REVIEW

**Comment by OGKevin** [comment]
@sandervdo please 👀 this one.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -7,9 +7,17 @@
 ## What happens:
 1.
 
-## Logs
-- Logs
+## Traceback
+[//]: # (If there is a traceback please share it in a quote! You can do this by pasting the traceback text, then highlight it and press the quote button.)
```
Comment (sandervdo) [inline .github/ISSUE_TEMPLATE.md]: You can do this by pasting the traceback text, then highlight it and press the quote button -> You can do this by pasting the traceback text, highlighting it and pressing the quote button.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -7,9 +7,17 @@
...
 
-## Extra info:
-- Tested on
+## SDK version and environment
+- Tested on [0.12.4](https://github.com/bunq/sdk_java/releases/tag/0.12.4)
```
Comment (sandervdo) [inline .github/ISSUE_TEMPLATE.md]: To save yourself some work, you might want to add a script in the future that automatically updates these numbers to the newest version. (see also [youtube-dl](https://github.com/rg3/youtube-dl/blob/master/.github/ISSUE_TEMPLATE_tmpl.md) as an example).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[//]: # (Thanks for opening this pull request! Before you proceed please make sure that you have an issue that explains what this pull request will do.
+         Make sure that all your commits link to this issue e.g. "My commit \(bunq/sdk_java#<issue nr>\)".
```
Comment (sandervdo) [inline .github/PULL_REQUEST_TEMPLATE.md]: Either make point in regex optional or make sure your example here matches it. 😄

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+         If this pull request is changing files that are located in "com.bunq.sdk.model.generated" then this pull request will be closed as these files must/can only be changed on bunq's side.)
+         
+## This PR closes/fixes the following issues:
+[//]: # (If for some reason your pull request does not require a test case you can just mark this box as checked and explain why it should not require a test case.)
```
Comment (sandervdo) [inline .github/PULL_REQUEST_TEMPLATE.md]: why it should not -> why it does not

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (OGKevin) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good one 😂

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (OGKevin) [reply .github/ISSUE_TEMPLATE.md]: These version number will be bumped during version bump at during the release phase

Comment (sandervdo) [reply .github/ISSUE_TEMPLATE.md]: youtube-dl also bumps the version number on a new release, but they have automated it. It's easy to forget one when the numbers appear in more places/examples. No biggy tho for now.

Comment (OGKevin) [reply .github/ISSUE_TEMPLATE.md]: Ah I see what you mean 😁 Will add this to my todo list! 😏

**Comment by OGKevin** [comment]
@sandervdo all yours 🐰

**Comment by OGKevin** [comment]
@andrederoos all yours 🌈

**Pull request merged**